graql/reasoner/negation.feature

```diff
@@ -541,6 +541,8 @@ Feature: Negation Resolution
   # MATCHING INFERRED CONCEPTS #
   ##############################
 
+  #  TODO: disabled because added new roles to existing relations no longer allowed in rule heads
+  @ignore
```

Comment:
Did we consciously decide to ignore these 3 tests, as opposed to just outright deleting them?

The scenarios being tested no longer make sense - they all specifically test negation against inferred attached roleplayers. If we're settled on banning roleplayer attachment in Reasoner, we should delete the scenarios. If we change our mind later, we can restore the scenarios from version control.

Reply:
Yup agreed, they've been deleted.